ui_tests/src/test/resources/com/saucedemo/account/signInNegative.feature

```diff
@@ -0,0 +1,68 @@
+
+
+Feature: Sign In
+
+
+  @signIn
+
+  Scenario: Sign in with empty username and password
+    Given  I open start page
+    When  I set ' ' as username on Sign in page
+    And   I set ' ' as password on Sign in page
```

Comment:
@MAKELAQA  In this: `I set ' ' as...` case you are setting a whitespace symbol ` ` (single space bar) as the value. Whitespace is a character (a symbol) i.e. it's not empty.
In order to set empty value to username or password you just keep the quotes empty, e.g. `I set '' as username on Sign in page`